hastexo/provider.py

```diff
@@ -1,20 +1,27 @@
 import time
-
+import base64
+import binascii
+import paramiko
+import random
+import string
+import yaml
+
+from StringIO import StringIO
```

Comment:
This only works in Python 2. Instead, use:

```python
try:
  from io import StringIO
except ImportError:
  from StringIO import StringIO
```

Reply:
Good catch!  Changes addressed in 765ad43.